lace/archive/nyx_archive.py

```diff
@@ -43,10 +43,11 @@ def __init__(self, file_name=None, kp_Mpc=0.7, verbose=False, zmax=None):
         # list all simulations
         self.list_sim = self.list_sim_cube + self.list_sim_test
         ## done set simulation list
-        
+
         # list of available redshifts at Nyx (not all sims have them)
-        self.list_sim_redshifts=np.append(np.arange(2.0,4.5,0.2),
-                np.arange(4.6,5.5,0.4))
+        self.list_sim_redshifts = np.append(
+            np.arange(2.0, 4.5, 0.2), np.arange(4.6, 5.5, 0.4)
```

Comment:
@lauracabayol changed this in another pull request, explicitly listing all redshifts because otherwise the bins where not exactly what she needed (I'm not sure why)

Reply:
Updated!